### DISCUSSION TITLE
Is it possible to group the R code for several pages of a website in one place to avoid repetition? I'm thinking of libraries and data reading?

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#8183] [Q&A]
### Description

Is it possible to group the R code for several pages of a website in one place to avoid repetition? I'm thinking of libraries and data reading?
I feel like I have to repeat the same code chunks  calling libraries and data on every page (R).

**Comment by mcanouil** [comment]
Duplicate:
- #8182
